common/models/api.js

```diff
@@ -0,0 +1,12 @@
+module.exports = function Api(Api) {
+  Api.remoteMethod(
+    'shutdown',
+    {
+      http: {path: '/shutdown', verb: 'post'},
+      isStatic: true,
+      accepts: [],
+      returns: {arg: 'response', type: 'object', root: true},
+      description: 'Shutdown this executor.'
```

Comment:
Changed to `Shutdown this server`. Intention is to shutdown whatever service is providing this API